lab03/src/Factorial.java

```diff
@@ -0,0 +1,22 @@
+public class Factorial {
+
+    public long recursionExecute(long n){
+        if (n < 2) {
+                return 1;
+            }
+        return recursionExecute(n - 1) * n;
+    }
+
+    public long cycleExecute(int n){
+        long result = 1;
+
+        for (int i = 1; i <= n; i ++){
+
```

Comment:
it is too much of empty rows.
I told about empty between logical blocks, but not in each row))